files/image_config/monit/container_checker

```diff
@@ -98,9 +114,12 @@ def main():
               and the containers which were expected to run. If containers which were exepcted
               to run were not running, then an alerting message will be written into syslog.
     """
-    expected_running_containers = get_expected_running_containers()
-    current_running_containers = get_current_running_containers()
+    expected_running_containers, always_running_containers = get_expected_running_containers()
+    ret, current_running_containers = get_current_running_from_DB(always_running_containers)
```

Comment:
One question: would it be feasible to combine the function `get_current_running_from_DB(...)` and `get_current_running_from_dockers()` together? and we can keep the function name as `get_current_running_containers()`.

Reply:
No big opinion in either way. If it would help with code read, which it sounds like, why not.

Reply:
Done.